loader/LoaderAndLayerInterface.md

```diff
@@ -946,15 +948,18 @@ Duplicate resulting folders will be ignored.
 An example combined search path for explicit layers may look like this:
 
 ```
+    /home/me/.config/vulkan/explicit_layer.d
+    /etc/xdg/vulkan/explicit_layer.d
     /usr/local/etc/vulkan/explicit_layer.d
-    /usr/local/share/vulkan/explicit_layer.d
     /etc/vulkan/explicit_layer.d
+    /home/me/.local/share/vulkan/explicit_layer.d
+    /usr/local/share/vulkan/explicit_layer.d
     /usr/share/vulkan/explicit_layer.d
-    $HOME/.local/share/vulkan/explicit_layer.d
 ```
 
-<b>IMPORTANT NOTE!</b> Any path using $HOME is ignored for all suid programs for
-security reasons.
+<b>IMPORTANT NOTE!</b> For security reasons, if the program using Vulkan
```

Comment:
Since $HOME isn't used in the example above, should we move this up to after line 937?